test/result-list-spec.js

```diff
@@ -1,38 +1,35 @@
 'use strict';
 
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-var mockagent = require('mockagent');
-var pixabay = require('../src/index');
-var promiseHelpers = require('promisehelpers');
-var range = require('lodash.range');
-var ResultList = require('../src/result-list');
-var sinon = require('sinon');
-var sinonChai = require('sinon-chai');
-var superagent = require('superagent');
-var url = require('url');
-
-var notify = promiseHelpers.notify;
-var wrap = promiseHelpers.wrap;
-
-var expect = chai.expect;
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const mockagent = require('mockagent');
+const pixabay = require('../src/index');
+const range = require('lodash.range');
+const ResultList = require('../src/result-list');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const superagent = require('superagent');
+const url = require('url');
+const {expect} = chai;
```

Comment:
Not sure how I feel about this, but I think it falls under personal opinion. I understand trying to avoid `require('...').whatever` but `something.whatever` seems fine to me. Again, personal opinion.